java/src/jmri/jmrit/vsdecoder/swing/ManageLocationsAction.java

```diff
@@ -131,4 +142,273 @@ public void actionPerformed(ActionEvent e) {
 
     private final static Logger log = LoggerFactory.getLogger(ManageLocationsAction.class);
 
+    /**
+     * class to serve as TableModel for Reporters and Blocks
+     */
+    static class ReporterBlockTableModel extends AbstractTableModel {
```

Comment:
I think it would be better to move this class to its own file, ReporterBlockTableModel.java, instead of moving it to the ManageLocationsAction class.

There is no reference to ReporterBlockTableModel in the ManageLocationsAction class, so I don't think ReporterBlockTableModel should belong to ManageLocationsAction.

Reply:
OK, will change it. Thanks!